tfbpmodeling/__main__.py

```diff
@@ -621,6 +621,8 @@ def test_sigmoid_interactor_significance(
     topn_sig_coefs = res.extract_significant_coefficients(ci_level=args.ci_level)
     logger.info("Top N Significant Coefs:" + str(topn_sig_coefs.keys()))
 
+    logger.info(f"Stage 4 method: {'LassoCV' if args.stage4_lasso else 'Linear'}")
```

Comment:
Rather than here, put a log statement to this effect in both the evaluate_interactor_significance_{linear/lassoCV} functions. That way, if the log statement from either function is present in the log, it guarantees that the execution took the right path.

It isn't a bad idea to log the parameters, but in this case I'm not very concerned about whether or not the flag works and more concerned that the code execution happens correctly.